docs/providers/aws/guide/serverless.yml.md

```diff
@@ -843,6 +843,8 @@ functions:
           # Set to 'true' when using an existing bucket
           # Else the bucket will be automatically created
           existing: true
+          # Optional, for forcing deployment of triggers on existing User Pools
```

Comment:
Let's make sure it does not reference User Pools in the comment - I believe it's a copy/paste comment that wasn't adjusted